subscriptions_test.go

```diff
@@ -797,6 +799,10 @@ func TestSubscriptions_UpdateNotes(t *testing.T) {
 	defer s.Close()
 
 	s.HandleFunc("PUT", "/v2/subscriptions/44f83d7cba354d5b84812419f923ea96/notes", func(w http.ResponseWriter, r *http.Request) {
+		body, _ := ioutil.ReadAll(r.Body)
+		if !strings.Contains(string(body), "gateway_code") {
+			t.Fatal("gateway code not encoded")
+		}
```

Comment:
This check could be problematic if, for example recurly ever added `gateway_codes` field. Since we know the format we should be more explicit in the checks (e.g. decoding xml in `r.Body` into `SubscriptionNotes` struct).

But since we already have a test format in the library for verifying encoding, let's revert these lines and use that instead. I didn't see a `TestSubscriptions_SubscriptionNotes_Encoding` test so it probably needs to be created as a top-level test. Here's a link to the test format for the `NewSubscription` struct as an example:

https://github.com/blacklightcms/recurly/blob/master/subscriptions_test.go#L17-L32

Reply:
thanks, done